elide-datastore/elide-datastore-aggregation/pom.xml

```diff
@@ -44,6 +44,11 @@
             <groupId>com.yahoo.elide</groupId>
             <artifactId>elide-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.yahoo.elide</groupId>
+            <artifactId>elide-datastore-hibernate</artifactId>
```

Comment:
I would rather see a dependency on the JPA store rather than hibernate.  The hibernate stores are probably going away in Elide 5.0